### ISSUE TITLE
In cnn-vgg16.ipynb, its acc has remained unchanged at 10%!

### ISSUE THREAD

**Issue by Totoro-wen** [rasbt/deeplearning-models#31]
The results are as follows.
```
# Epoch: 001/010 | Train: 10.000% | Loss: 2.520%
# Time elapsed: 2.24 min
# Epoch: 010/010 | Train: 10.000% | Loss: 2.303%
# Time elapsed: 22.42 min
# Total Training Time: 22.42 min
# Test accuracy: 10.00%
```
 I have tried to increase the learning rate by adding Dropout and nn.Adaptive eavgpool2d, but acc has remained unchanged!
Thanks!

**Comment by rasbt** [comment]
Hm, that's weird. I just reran that code in PyTorch 1.3 and it still works fine (pushed the changes to GitHub). What PyTorch version are you on?

**Comment by Totoro-wen** [comment]
> Hm, that's weird. I just reran that code in PyTorch 1.3 and it still works fine (pushed the changes to GitHub). What PyTorch version are you on?

Hello,my pytorch version  is 1.2.

**Comment by rasbt** [comment]
hm that's weird. Maybe you accidentally changed something in the notebook. Could you try to redownload it and run it again?

**Comment by Totoro-wen** [comment]
> hm that's weird. Maybe you accidentally changed something in the notebook. Could you try to redownload it and run it again?

Thanks.

**Comment by Totoro-wen** [comment]
> hm that's weird. Maybe you accidentally changed something in the notebook. Could you try to redownload it and run it again?

After I redownloaded it and ran it, it worked.
Thank you very much.

**Comment by rasbt** [comment]
Nice! Glad to hear that it solved the problem.

**Comment by annyWangAn** [comment]
I have the same problem. I redownloaded it and ran it. But it didn't work. My pytorch version is 1.2. The loss didn't decline. The acc has remained unchanged.
